quickstart/aspnet/ContosoTeamStats/Web.config

```diff
@@ -1,55 +1,148 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <!--
   For more information on how to configure your ASP.NET application, please visit
   https://go.microsoft.com/fwlink/?LinkId=301880
   -->
 <configuration>
   <appSettings file="C:\AppSecrets\CacheSecrets.config">
-    <add key="webpages:Version" value="3.0.0.0"/>
-    <add key="webpages:Enabled" value="false"/>
-    <add key="ClientValidationEnabled" value="true"/>
-    <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
+    <add key="webpages:Version" value="3.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="ClientValidationEnabled" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+	<add key="RedisHostName" value="your_redis_hostname"/>
   </appSettings>
+  <!--
```

Comment:
Was leaving this comment here intentional?

Reply:
The comments were not added by me. I took the web.config as it was and added the changes I needed. I can delete it if needed? @stanleysmall-microsoft